ReactiveObjC/RACSignal.h

```diff
@@ -45,13 +45,13 @@ NS_ASSUME_NONNULL_BEGIN
 /// subscribes. Any side effects within the block will thus execute once for each
 /// subscription, not necessarily on one thread, and possibly even
 /// simultaneously!
-+ (RACSignal *)createSignal:(RACDisposable * _Nullable (^)(id<RACSubscriber> subscriber))didSubscribe;
++ (instancetype)createSignal:(RACDisposable * _Nullable (^)(id<RACSubscriber> subscriber))didSubscribe;
```

Comment:
I think all the signal methods are incorrect as `instancetype`. Since they're implemented as a class cluster, where the returned object is one of a number of private implementation classes, `instancetype` wouldn't be correct for any subclasses.